pe.c

```diff
@@ -873,9 +873,9 @@ handle_sbat(char *SBATBase, size_t SBATSize)
 	return efi_status;
 }
 
-static EFI_STATUS verify_image(void *data, unsigned int datasize,
-			       EFI_LOADED_IMAGE *li,
-			       PE_COFF_LOADER_IMAGE_CONTEXT *context)
+EFI_STATUS verify_image(void *data, unsigned int datasize,
```

Comment:
This function was introduced in a previous commit - seems better to fold this change (and the header addition) into that